NEWS.md

```diff
@@ -1,5 +1,19 @@
 # ggplot2 (development version)
 
+<<<<<<< HEAD
```

Comment:
There is something here🙂

Reply:
Why yes, yes there is. 😂 I feel like this is the second time that my commit message should have a facepalm emoji.  

I actually searched the project for <<<<, but I realize now I was only searching through the R script files... Stay tuned. Thanks!